packages/webui/src/client/ui/Settings/ShowStyleBaseSettings.tsx

```diff
@@ -149,6 +150,7 @@ export default translateWithTracker<IProps, IState, ITrackedProps>((props: IProp
 									</Route>
 									<Route path={`${this.props.match.path}/action-triggers`}>
 										<TriggeredActionsEditor
+											studioId={protectString('fakeStudio')} // A studioId is needed to enable the preview renders
```

Comment:
This smells of tech dept to me. I see that `null` is allowed in types, would that work instead?

Reply:
Yeah, I can see that I'm the author of that change, but can't remember ever made it.
So it's probably some old stuff that got reinserted when we merged back and forth.
@Julusian du you have a clue about it?

Reply:
no, `null` does not allow the previews to display.

I wrote a bunch of explanation for this, then realised that we could just take the `studioId` from the `rundownPlaylist` that we know we already have slightly later on. Which makes things simpler and more correct.

~~This is because it gets checked for being truthy https://github.com/bbc/sofie-core/blob/e839b97c0205df38e5bc7ab816a9c79406a7d767/packages/webui/src/client/ui/Settings/components/triggeredActions/TriggeredActionsEditor.tsx#L395
because the property is not allowed to be null in the context type which this is used to build https://github.com/bbc/sofie-core/blob/e839b97c0205df38e5bc7ab816a9c79406a7d767/packages/meteor-lib/src/triggers/actionFactory.ts#L39~~

~~Which it turns out isn't hard to handle, that property is only fetched in one place, but it does feel odd that a `ReactivePlaylistActionContext` can know what playlist and rundown is being played, but not have a studio. Not a big issue for now though~~

~~That said, I have noticed that the ui hasn't been built to actually choose the routeset to switch. I expect that will need to know what studio it should be showing, as it needs to pick an id.~~